package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This shouldn't exist at the top level. These dependency additions belong in the package.json that exists in the web-ui project.